botfront/imports/startup/server/apollo.js

```diff
@@ -51,7 +51,12 @@ export const runAppolloServer = () => {
     });
 
     WebApp.connectHandlers.use('/health', (req, res) => {
-        axios.get('http://localhost:3000/graphql?query=query%20%7BhealthCheck%7D').then((response) => {
+        const { authorization } = req.headers;
+        axios.get('http://localhost:3000/graphql?query=query%20%7BhealthCheck%7D', {
+            headers: {
```

Comment:
Shouldn't we handle the case where authorization is undefined and just not set headers in that case?

Reply:
Yes, I've made the change